extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -171,8 +170,11 @@ export class AmpConsent extends AMP.BaseElement {
         () => this.handleAction_(ACTION_TYPE.DISMISS));
     this.registerAction('prompt', invocation => {
       const args = invocation.args;
-      const consentId = args && args['consent'];
-      this.handlePostPrompt_(consentId);
+      let consentId = args && args['consent'];
+      if (!this.isMultiSupported_) {
+        consentId = Object.keys(this.consentConfig_)[0];
```

Comment:
This is just getting a random key, rather than the first. Is that ok?

Reply:
That's OK, there's only one key anyway.